### DISCUSSION TITLE
Courseplay on Geforce Now

### DISCUSSION THREAD

**Discussion by Elo9998** [Courseplay/Courseplay_FS22#3270] [Q&A]
Hi, 
is there any possibility to play FS 22 on Geforce Now with Courseplay?

Greetings

**Comment by Tensuko** [comment]
If geforce now supports the modhub, it should.
Why would it not work?

**Comment by Elo9998** [comment]
The modhub is supported but i cant find Courseplay in there. It seems like the are just not supporting Courseplay.

**Reply by Tensuko** [reply]
Are there any other script mods?
If no, then this is the reason. (like console)
